test/services/actionRulesService.test.js

```diff
@@ -0,0 +1,37 @@
+const actionRulesService = require('../../server/services/actionRulesService')
+const actionsService = require('../../server/services/actionsService')
+const actionID = 'FG1'
+const ruleID1 = 3
+const ruleID2 = 5
+const enabled = false
+
+describe('actionRulesService', () => {
+  test('action rules service returns an object', async () => {
+    const rule = await actionRulesService.update(actionID, ruleID1, enabled)
+    expect((rule instanceof Object)).toEqual(true)
```

Comment:
I think a more Jesty way to write this assertion is: `expect(rule).toEqual(expect.any(Object))`

https://jestjs.io/docs/en/expect#expectanyconstructor

Reply:
Test has been updated.